libs/design-system/src/molecules/alerts/alerts.tsx

```diff
@@ -56,7 +65,7 @@ const Alert = ({ type = 'info', className, link, message }: AlertsProps) => {
     </div>
   );
   return link ? (
-    <a href={link} target="_blank" rel="noreferrer">
+    <a href={link} target="_blank" rel="noreferrer" className=" ">
```

Comment:
Why this className?

Reply:
Because I was tired and forgot to remove it when testing stuff 😅